app/models.py

```diff
@@ -1,35 +1,56 @@
-from app import db
-# from sqlalchemy.orm import relationship
+from app import db, login
+from datetime import datetime, timezone
+from werkzeug.security import check_password_hash
+from flask_login import UserMixin
 
 
+@login.user_loader
+def load_user(id):
+    return User.query.get(int(id))
 
-class User(db.Model):
+
+class User(UserMixin, db.Model):
+    __tablename__ = 'users'
     id = db.Column(db.Integer, primary_key=True)
-    username = db.Column(db.String(40), unique=True, nullable=False)
-    password = db.Column(db.String(128), nullable=False)
+    # indexing the username and email columns for faster search since will be used for login
+    username = db.Column(db.String(64), unique=True, nullable=False, index=True)
+    email = db.Column(db.String(120), unique=True, nullable=False, index=True)
+    # to store profile picture
+    image_file = db.Column(db.String(20), nullable=False, default='default.jpg')
+    password = db.Column(db.String(256), nullable=False)
+    # for short bio
+    about = db.Column(db.String(256), default='Hello, I am new here!')
+    # To know when the account was created for requestee and account page
+    account_created = db.Column(db.DateTime, nullable=False, default=datetime.now(timezone.utc))
     ratings = db.Column(db.Float(5.0), default=0.0)
     ratings_count = db.Column(db.Integer(), default=0)
-    # author = db.relationship('Post', backref='posts', lazy='dyanmic')
-
```

Comment:
UserMixin class already implements these so if i inherit from it no need to explicitly define them.